app/assets/stylesheets/forever_style_guide/modules/_hero-blur.scss

```diff
@@ -0,0 +1,147 @@
+.hero-blur {
+  height: 100%;
+  opacity: 0;
+  background-position: top center;
+  background-repeat: no-repeat;
+  @include clearfix;
+  @include transition-property(opacity);
+  @include transition-duration(1s);
+  @include transition-timing-function(ease-in);
+}
+
+.hero-blur-block {
+  position: relative;
+  margin-top: 150px;
+  opacity: 0;
+  @include box-shadow(0 0 4px 2px rgba(color('gray'), 0.25));
+  @include transition-property(opacity);
+  @include transition-duration(1s);
+  @include transition-timing-function(ease-in);
+
+  @media (min-width: $screen-lg) {
+    margin-top: 250px;
+    margin-bottom: 50px;
+  }
+  @media(min-width: $screen-xl) {
+    margin-top: 300px;
+    margin-bottom: 100px;
+  }
+}
+.hero-blur-block-visible,
+.hero-blur-visible {
+  opacity: 1.0;
+}
+
+.hero-block {
```

Comment:
strikes me that some of these might want to be refactored into the hero module or perhaps its own hero-block sub-module